charts/portworx/values.yaml

```diff
@@ -21,15 +21,15 @@ managementInterface: none             # Name of the interface <ethX>
 envVars: none                         # NOTE: This is a ";" seperated list of environment variables. For eg: MYENV1=myvalue1;MYENV2=myvalue2
 
 stork: true                           # Use Stork https://docs.portworx.com/scheduler/kubernetes/stork.html for hyperconvergence.
-storkVersion: 2.3.1
+storkVersion: 2.3.3
 
 customRegistryURL:
 registrySecret:
 
 lighthouse: true
-lighthouseVersion: 2.0.5
-lighthouseSyncVersion: 2.0.5
-lighthouseStorkConnectorVersion: 2.0.5
+lighthouseVersion: 2.0.6
+lighthouseSyncVersion: 2.0.6
+lighthouseStorkConnectorVersion: 2.0.6
```

Comment:
all of these are 2.0.7 now. I compared with spec returned by current https://install.portworx.com/?comp=lighthouse